pkg/agent/dozer/bcm/plan.go

```diff
@@ -883,6 +883,34 @@ func planExternals(agent *agentapi.Agent, spec *dozer.Spec) error {
 				Action: dozer.SpecPrefixListActionPermit,
 			}
 		}
+
+		spec.RouteMaps[ipnsSubnetsRouteMapName(ipnsName)] = &dozer.SpecRouteMap{
+			Statements: map[string]*dozer.SpecRouteMapStatement{
+				"10": {
+					Conditions: dozer.SpecRouteMapConditions{
+						MatchPrefixList: pointer.To(ipnsSubnetsPrefixListName(ipnsName)),
+					},
+					Result: dozer.SpecRouteMapResultAccept,
+				},
+			},
+		}
+
+		aclName := ipNsNoExtPeeringACLName(ipnsName)
+		entries := map[uint32]*dozer.SpecACLEntry{}
+		for idx, subnet := range ipns.Subnets {
+			entries[uint32(idx+1)] = &dozer.SpecACLEntry{ //nolint:gosec
```

Comment:
Generally, we're trying to avoid situations where adding another subnet at the beginning of the IPNS subnet list would cause all entries to be rewritten. The Catalog's subnetIDs should be used instead. We already allocate IDs for some things like VPC subnets, etc, so please just add IPNS subnets to it as well (`subnetsReq` in agent ctrl).